Documentation/ApiOverview/Backend/JavaScript/ES6/Index.rst

```diff
@@ -91,6 +95,27 @@ with the following statement:
 
     import $ from 'jquery';
 
+Add JavaScript modules to importmap in backend form
```

Comment:
Below "importmap" is written as "import map". We should stay consistent.

Reply:
The specification calls them import maps https://html.spec.whatwg.org/multipage/webappapis.html#import-maps